src/bosh_azure_cpi/lib/cloud/azure.rb

```diff
@@ -51,31 +51,38 @@ module AzureCloud; end
 require 'cloud/azure/models/bosh_vm_meta'
 require 'cloud/azure/utils/command_runner'
 require 'cloud/azure/models/error_msg'
+require 'cloud/azure/models/object_id_keys'
 require 'cloud/azure/models/config'
 require 'cloud/azure/models/config_factory'
 require 'cloud/azure/models/object_id'
 require 'cloud/azure/models/res_object_id'
 require 'cloud/azure/models/disk_id'
 require 'cloud/azure/models/ephemeral_disk'
-require 'cloud/azure/models/instance_id'
+require 'cloud/azure/models/vm_instance_id'
+require 'cloud/azure/models/vmss_instance_id'
 require 'cloud/azure/models/props_factory'
 require 'cloud/azure/models/root_disk'
 require 'cloud/azure/models/vm_cloud_props'
+require 'cloud/azure/models/cloud_id_parser'
+require 'cloud/azure/models/vmss_config'
 require 'cloud/azure/cloud'
+require 'cloud/azure/restapi/rest_helpers'
```

Comment:
better to follow alphabet order?

Reply:
azure_client depend on rest_helpers.
ruby require rest_helpers before azure_client.
lovely script language.